e2e/fixtures/ccd/expectedEvents.js

```diff
@@ -4,21 +4,26 @@ module.exports = {
   applicant_solicitor: {
```

Comment:
Please can you create a tech-debt ticket in the backlog to add this event into the end-to-end tests that we have.

Reply:
This shouldn't be a tech debt ticket - the tests should be updated as part of this ticket.
Please add this to to the API and e2e tests for 1v2 diff sols please